feature-detects/hbbtv/video.js

```diff
@@ -0,0 +1,524 @@
+/* global setTimeout */
+/*!
+{
+  "name": "HbbTV Video",
+  "property": "hbbtvvideo",
+  "tags": ["hbbtv", "oipf", "video"],
+  "async" : true,
+  "warnings": [
+    "This test is currently only compatible to HbbTV >= 1.x",
+    "All siblings of an existing HbbTV video object in the DOM will be deleted during the test"
```

Comment:
What happen if there is more than one video?

you should be able to `cloneNode()` on the video element, cache that, and then readd it later, no?